src/Kunstmaan/MediaBundle/Helper/File/FileHandler.php

```diff
@@ -108,20 +108,22 @@ public function getFormType()
         return FileType::class;
     }
 
-    /**
-     * @param mixed $object
-     *
-     * @return bool
-     */
-    public function canHandle($object)
+    public function canHandle($object): bool
     {
-        if ($object instanceof File ||
-            ($object instanceof Media &&
-            (is_file($object->getContent()) || $object->getLocation() == 'local'))
-        ) {
+        if($object instanceof File) {
             return true;
         }
 
+        if($object instanceof Media) {
+            if($object->getLocation() === 'local') {
+                return true;
+            }
+
+            if($object->getContent() !== null && is_file($object->getContent())) {
+                return true;
+            }
+        }
+
```

Comment:
```suggestion
        if (!$object instanceof Media) {
            return false;
        }
        
        if ($object->getLocation() === 'local') {
            return true;
        }

        if ($object->getContent() !== null && is_file($object->getContent())) {
            return true;
        }
```

1 less level of nesting, makes it easier to read